lms/static/js/commerce/track_ecommerce_events.js

```diff
@@ -0,0 +1,40 @@
+/**
```

Comment:
I ended up not adding this to any other config files and it loaded ok so I'm guessing this path is already auto-added/loaded into the correct scope (it works locally). Happy to move this to somewhere else, rename, etc. as needed.

Reply:
Also if anyone has ideas on how to test this via unit tests, I'm all ears :)